platform/ios/scripts/document.sh

```diff
@@ -44,4 +44,4 @@ jazzy \
     --output ${OUTPUT}
 # https://github.com/realm/jazzy/issues/411
 find ${OUTPUT} -name *.html -exec \
-    perl -pi -e 's/Mapbox\s+(Docs|Reference)/Mapbox iOS SDK $1/' {} \;
+    perl -pi -e 's/BRANDLESS_DOCSET_TITLE/iOS SDK $1/' {} \;
```

Comment:
We still need the old replacement as well because of places outside the header that end up saying “Mapbox Docs” or “Mapbox References” otherwise.